package.json

```diff
@@ -7,7 +7,7 @@
     "build:electron": "cross-env NODE_ENV=production --target=electron-renderer webpack --mode=production",
     "build:electron:pixi": "cross-env NODE_ENV=production --graphics=pixi --target=electron-renderer webpack --mode=production",
     "build:web": "webpack --target=web --mode=production",
-    "build:web:pixi": "cross-env --graphics=pixi webpack --mode=production",
+    "build:web:pixi": "cross-env --graphics=pixi webpack --target=web --mode=production",
```

Comment:
somehow missed this `--target` declaration. Otherwise, `npm run build:web:pixi` puts everything into the `dist/electron` folder. Oops.